### DISCUSSION TITLE
Getting White screen after deploying static web app

### DISCUSSION THREAD

**Discussion by sanskarkakde14** [microsoft/SATechnicalOnboarding#40] [Questions]
Getting White screen after deploying static web app even workflow actions are healty
<img width="1440" alt="Screenshot 2023-06-13 at 9 12 13 PM" src="https://github.com/microsoft/SATechnicalOnboarding/assets/80622561/e9552a2d-4664-4106-872a-cef6cf8b04da">

**Comment by seths10** [comment]
what do you mean by white screen? also, the screenshot doesn't show a white screen

**Comment by sanskarkakde14** [comment]
thank u lord for replying so quickly

**Comment by seths10** [comment]
where do you get the whitescreen?

**Reply by rahmlad-aramide** [reply]
I believe he's referring to his deployed project when viewed using the deployed URL, it's displaying a blank screen

**Comment by rahmlad-aramide** [comment]
If you're still having the issue and you're probably trying to deploy on github pages, here's a step by step guide which I believe should solve the problem from the prompt I get from ChatGPT: https://chat.openai.com/share/5607fd9c-8cea-42de-876a-93e9a0a8a6c9
